apps/core/src/hooks/useTimeAgo.ts

```diff
@@ -128,7 +128,7 @@ export const timeAgo = (
 type Format = 'year' | 'month' | 'day' | 'hour' | 'minute' | 'second' | 'weekday';
 
 export function formatDate(date: Date | number, format?: Format[]): string {
```

Comment:
we might as well clean it up
```suggestion
export function formatDate(date: Date | number, format: Format[] = ['day', 'month', 'hour', 'minute'] ): string {
```